pkgs/development/python-modules/tomlkit/default.nix

```diff
@@ -10,11 +10,14 @@
 
 buildPythonPackage rec {
   pname = "tomlkit";
-  version = "0.11.6";
+  # we have pinned tomlkit to 0.11.4 as newer versions are no longer including
+  # setup.py (which is required to build tomlkit without poetry).
+  # see upstream issue: https://github.com/sdispater/tomlkit/issues/305
+  version = "0.11.4";
```

Comment:
We downgrade to <0.11.6 because of the changed semantics in versions 0.11.6 to [are the semantics fixed in 0.11.7 or 0.11.8?], and we don't upgrade to 0.11.8 because of the `setup.py` issue.

We could also try just downgrade to 0.11.5, there's nothing wrong with that version afaik.

Reply:
I feel like upgrading to 0.11.8 with a patch would be best. Are you opposed to that?